### PR TITLE
Append  res-life to blossom-ci whitelist [skip ci]

### PR DESCRIPTION
**Pull request by res-life**
Signed-off-by: Chong Gao <res_life@163.com>

### CODE REVIEW

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ jobs:
       ttnghia,\
       zhanga5,\
       nvliyuan,\
+      chongg,\
```
Comment (pxLi) [inline .github/workflows/blossom-ci.yml:64]: this should be your github account ID

**Review by pxLi — CHANGES_REQUESTED** [review]

**Comment by pxLi** [comment]
why you change the title? all PRs will be added to the change log when release, there should be no personal info here

File: .github/workflows/blossom-ci.yml (hunk 1 above)
Comment (res-life) [reply .github/workflows/blossom-ci.yml]: already changed to my github id

**Comment by pxLi** [comment]
build

**Review by pxLi — APPROVED** [review]

**Pull request merged**